dependencies/pom.xml

```diff
@@ -1021,6 +1021,28 @@
                 <artifactId>neo4j-java-driver</artifactId>
                 <version>${version.lib.neo4j}</version>
             </dependency>
+            <dependency>
```

Comment:
Two things:
1. I think these exclusions should be done on the dependency that is bringing in the undesired dependency. So in this case on `microprofile-health-api` and `microprofile-metrics-api`
2. In general io.helidon dependencies should be managed in the bom, not here.

Reply:
@tomas-langer proposed to do it in dependancies/pom.xml
my first intention was to do it in bom/pom.xml

Reply:
If you move the exclusions to `microprofile-health-api` and `microprofile-metrics-api` then `dependencies/pom.xml` is definitely the right place.

If you must do the exclusions on helidon artifacts then `bom/pom.xml` seems more appropriate to me (unless I'm missing something which is possible!)

Reply:
Ah, got it. Now moved to the right place.